code/index.html

```diff
@@ -1,22 +1,168 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
-
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>News Magazine</title> 
+    <link rel="stylesheet" href="style.css">
+    <link rel="icon" href="./news.png">     <!--favicon-->
+    <!--fonts from Google-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Kaisei+Opti:wght@400;500;700&display=swap" rel="stylesheet">
+ </head>
+    <body>
+        <!--top-bar with two div, top-bar left and top-bar right-->
+        <nav class="top-bar" id="top">
+            <section class="top-bar left">
+                <div class="logo">
+                    <img src="./news.png" 
+                    alt="news icon" 
+                    width="30" height=30>
+                </div>
+
+                <div class="search">   
+                    <a href="https://www.google.com">  
+                        <img src="./search.png"   
+                        alt="search icon" 
+                        width="30" height=30>
+                    </a>
+                </div>
+
+                <div class="contact">
+                    <a href="#bottom"> <h5>contact us</h5> </a>
+                </div>
+            </section>  
+
+            <section class="top-bar right">     <!--made a link wrapping around each social media icon-->
+                <a href="https://www.facebook.com" target="blank"> <!-- target ="blank" -> new tab-->                 
```

Comment:
I really like that you make small comments on sections, explaning what the code does! Cudos!!